internal/runner/options.go

```diff
@@ -65,6 +66,7 @@ func ParseOptions() *Options {
 		flagSet.StringSliceVar(&options.Id, "id", nil, "display results for given ids (comma-separated)", goflags.NormalizedStringSliceOptions),
 		flagSet.BoolVar(&options.Hosts, "host", false, "display only hostnames in results"),
 		flagSet.BoolVar(&options.IPAddress, "ip", false, "display only ips in results"),
+		flagSet.StringSliceVarP(&options.Services, "service", "s", nil, "query and display results from given service (comma-separated))", goflags.NormalizedStringSliceOptions),
```

Comment:
I believe, we can use enumslicevarp instead https://github.com/projectdiscovery/goflags/blob/main/examples/basic/main.go#L45

Reply:
`enumslicevarp` can be used for provider option as well. cc @RamanaReddy0M